tox.ini

```diff
@@ -40,3 +36,9 @@ commands =
     rm -rf .pytest_cache/
     rm -rf nim.egg-info/
 whitelist_externals = rm
+
+[travis]
+python =
+    3.6: py36, flake8, pylint, test, bandit
+    3.7-dev: py37-dev, test
+    pypy3: pypy3, test
```

Comment:
This is to tell Travis to run `test` on all environments, but only run `flake8`, `pylint` and `bandit` on Python 3.6 as it's the version we support and running those on other versions of Python should work the same way, so this makes the build faster.